packages/status-js/src/preview-client/map-user-preview.ts

```diff
@@ -0,0 +1,38 @@
+import type { ContactCodeAdvertisement } from '../proto/communities/v1/push_notifications'
+
+export type UserPreview = {
+  photo?: Uint8Array
+  displayName: string
+  description?: string
+  // todo!:
+  // emojiHash: string
+  socialUrls: Array<{
+    url: string
+    // todo?: map fixed (e.g. telegram) and custom
```

Comment:
URLs can be of custom value too. And if this mapper were to be used elsewhere similarly to https://github.com/status-im/status-web/pull/342#discussion_r1103295001, it might be worth mapping those urls which actually have a handle in them and as follows:

For

```
{text: '__personal_site', url: 'https://example.com'}
{text: '__github', url: 'https://github.com/example'}
...
```

like

```
{type: '__personal_site', url: 'https://example.com'}
{type: 'github', url: 'https://github.com/example', handle: 'example'}
```

Reply:
Agreed.

Reply:
Social links were removed in latest designs, see https://www.figma.com/file/Nt3KJ8AXm0VPpIRLxaHnw0?node-id=2339:314624#389950815.

Reply:
Removed at https://github.com/status-im/status-web/pull/342/commits/3d55457fa65dfe7552f9e085e6d10759b35bc2ac.